src/main/resources/avro/references.avdl

```diff
@@ -81,21 +81,12 @@ record ReferenceSet {
   string id;
 
   /**
-  The IDs of the `Reference` objects that are part of this set.
-
-  This field should be left null if the set contains a very large number of
-  `Reference`s. The `Reference`s in the set may also be obtained through a
```

Comment:
>  I think depending on divergent sets of documentation in language-specific client libraries being better is not a good solution - in my opinion an ideal client library would be very thin and use as much autogenerated code from this schemas repo as possible (including docs).

Yeah, you're right. I suppose what we're lacking is the 'big picture' documentation, but I think that's underway at UCSC.

I'd forgotten we'd talked about removing this ``referenceIds`` thing before: we should have made a PR on the spot!